.github/workflows/node.js.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master, release/*, chore/* ]
+  pull_request:
+    branches: [ master , release/*]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 13.x, 14.x]
```

Comment:
Let's remove 13.x IMO. Non-prod version for 6.0 we should be at 12 or hopefully 14 before release. This should save a little bit of github actions time.

Reply:
removed 13.x

Reply:
```suggestion
        node-version: [10.x, 12.x, 14.x]
```